docs/excel/excel-add-ins-tables.md

```diff
@@ -160,6 +160,37 @@ Excel.run(function (context) {
 
 ![Table with new calculated column in Excel](../images/excel-tables-add-calculated-column.png)
 
+## Resize a table (preview) (online-only)
+
+> [!NOTE]
+> The `Table.resize` method is currently available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)].
```

Comment:
Since this is in ExcelApiOnline, it is not in preview and should not have this note. Developers could include this API in production add-ins.

Reply:
@ElizabethSamuel-MSFT We discussed offline, and the plan is that I will identify this API as "online-only" here. In the online-only article (to be potentially updated in a subsequent PR), I will consider adding a section that indicates which online-only APIs are candidates for the upcoming GA release.